1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java

```diff
@@ -48,7 +49,10 @@ private void init(CallbackInfo callbackInfo) {
 
     @Inject(method = "keyTyped", at = @At("RETURN"))
     private void updateLength(CallbackInfo callbackInfo) {
-        if (inputField.getText().startsWith(String.valueOf(LiquidBounce.commandManager.getPrefix())) && !inputField.getText().startsWith(LiquidBounce.commandManager.getPrefix() + "lc"))
+        if (!inputField.getText().startsWith(String.valueOf(LiquidBounce.commandManager.getPrefix()))) return;
+        LiquidBounce.commandManager.autoComplete(inputField.getText());
+
+        if (!inputField.getText().startsWith(LiquidBounce.commandManager.getPrefix() + "lc"))
```

Comment:
What was fixed here?

Reply:
https://github.com/CCBlueX/LiquidBounce1.8-Issues/issues/3795
Watch the Video. In the current build the completion will write the second command suggestion, not the first one.